### PR TITLE
fix: update page title w/ i18n

### LINKED ISSUE #5: i18n into HEAD to get title to update

### PR DESCRIPTION
**Pull request by vim-usds**
- Head API cannot support i18m
- Replace Head API with react-helmet
- closes #5

### CODE REVIEW

File: src/components/SEO/SEO.tsx [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+
+      {/* Description content should ideally be between 160 - 200 characters */}
+      <meta
+        name="description"
+        content={`This new web app will do X for the government providing Y, Z and W.`}
+      />
+
+      {/* Keyword content should list the top 10 words in order of relevance. The words chosen need
+        to appear on every page. These words were chosen from the header as it appears on each page */}
+      <meta
```
Comment (mattbowen-usds) [inline src/components/SEO/SEO.tsx:27]: i'm curious why you include meta keywords at all?

File: src/components/SEO/SEO.tsx [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
-      {/* Document Title */}
-      <title>{title}</title>
-
-      {/* Open Graph Meta Tags (ogp.me) */}
-      <meta property="og:site_name" content="Site Name" />
-      <meta property="og:title" content="Page Title" />
-      <meta property="og:description" content="Description of this web page" />
-    </>
+    <Helmet defer={false}>
+      <title>{`${title}`}</title>
```
Comment (mattbowen-usds) [inline src/components/SEO/SEO.tsx:17]: Did you mean to drop the charset and httpEquiv?

**Review by mattbowen-usds — APPROVED** [review]
I have one question that wouldn't block merge.

File: src/components/SEO/SEO.tsx (hunk 1 above)
Comment (vim-usds) [reply src/components/SEO/SEO.tsx:27]: I was thinking that they helped out for SEO?

File: src/components/SEO/SEO.tsx (hunk 2 above)
Comment (vim-usds) [reply src/components/SEO/SEO.tsx:17]: Nope! Nice one!

File: src/components/SEO/SEO.tsx (hunk 1 above)
Comment (mattbowen-usds) [reply src/components/SEO/SEO.tsx:32]: Nah, they don't. They're too easy to game, so search engines ignore them. I'd omit as to not confuse people

**Pull request merged**